docs/guides/app-dev/indexing-transactions.md

```diff
@@ -142,6 +142,21 @@ Example:
 psql ... -f state/indexer/sink/psql/schema.sql
```

Comment:
Should we make some changes to this schema as well?

Reply:
I wasn't sure how y'all would like that done? The way I used it in my project is I copied it over into my own postgres migrations so I was able to modify the table names and add my own `down` migration as well. I assume most people that would need custom table names would do it that way as well? I noted what to do in the documentation here.
https://github.com/cometbft/cometbft/pull/3594/files#diff-87febb1abff3a71169dde69e07cc7d4a2bc49b3111a5252faab9d0b5a6979a0bR146-R148

Reply:
I see. I am really not an expert on SQL, but do you know any way to render the table names parametrizable?

Reply:
Given that it's a static file and seems to be used as such, I'm not aware of a way to make it parametrizable. The way to do that would probably be to have the `psql` indexer run migrations on it's own instead of expecting the tables to already exist.